src/cli/cli.cpp

```diff
@@ -4615,12 +4615,11 @@ otError Interpreter::ProcessDiag(uint8_t aArgsLength, Arg aArgs[])
     char    output[OPENTHREAD_CONFIG_DIAG_OUTPUT_BUFFER_SIZE];
 
     // all diagnostics related features are processed within diagnostics module
-    output[0]                  = '\0';
-    output[sizeof(output) - 1] = '\0';
+    output[0] = '\0';
```

Comment:
This seems unnecessary according to `otDiagProcessCmd`'s implementation?

Reply:
Make sense to me.